app/services/invoices/payments/stripe_service.rb

```diff
@@ -137,7 +139,14 @@ def stripe_payment_payload
       def update_invoice_payment_status(payment_status)
         return unless Invoice::PAYMENT_STATUS.include?(payment_status&.to_sym)
 
-        invoice.update!(payment_status: payment_status)
+        invoice.update!(
+          payment_status: payment_status,
+          ready_for_payment_processing: payment_status.to_s == 'failed'
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)